features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2450,25 +2450,6 @@ REFERENCING NEW AS NEW FOR EACH ROW MODE DB2SQL
         SET (NEW.ID) = (NEXTVAL FOR IDN_WEBHOOK_TOPIC_SEQ);
     END
 /
-CREATE TABLE IDN_WEBHOOK_METADATA (
```

Comment:
Shall we keep the schema as we are not removing the application logic to persist data to these tables

Reply:
Invocation parts are removed. Therefore the schema is not needed.